adapter-mongodb/src/main/resources/activities/mongodb-basic-uuid.yaml

```diff
@@ -1,10 +1,10 @@
-# nb -v run driver=mongodb yaml=mongodb-basic-uuid connection=mongodb://127.0.0.1 database=testdb tags=phase:rampup
+# nb -v run driver=mongodb yaml=mongodb-basic-uuid connection=mongodb://127.0.0.1 database=testdb tags=block:rampup
 description: An example of a basic mongo insert and find with UUID
 
 scenarios:
   default:
-    - run driver=mongodb tags==phase:rampup cycles===TEMPLATE(rampup-cycles,100000000) threads=auto
-    - run driver=mongodb tags==phase:main cycles===TEMPLATE(main-cycles,100000000) threads=auto
+    - run driver=mongodb tags==block:rampup cycles===TEMPLATE(rampup-cycles,100000000) threads=auto
```

Comment:
Named scenarios with _list_ structure should be changed to _map_ structure with step names.

Reply:
@jshook I wasn't originally including the `mongodb` changes in this first-pass as it was becoming unwieldy to verify all of these changes.  However, thinking about it, I'll start including all the adapter changes in this PR so we can get them in one chunk.  I will most likely need another set of 👀 for verifications and what is no longer relevant (if any).  wdyt?  I'll add the list I'm tracking to this PR and if you can identify that are no longer relevant that will help prune a few of these.

Reply:
Added mongodb.